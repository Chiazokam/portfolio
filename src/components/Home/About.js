import React from 'react';
import Container from '@material-ui/core/Container';

const About = () => {
    return (
        <section className='about-section'>
            <Container fixed>
                <div className='section-title'>Who I am...</div>
                <div className='about-content'>
                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut quis sapien diam. Sed sagittis, sapien eget porta 
                    semper, sem sem imperdiet augue, eget elementum enim turpis a lorem. Praesent id nisi odio. Aliquam commodo 
                    nibh vitae quam pharetra ultricies. Praesent et vehicula ante. Nunc at erat libero. Nullam quis cursus nisl. Maecenas 
                    placerat auctor leo, vel maximus orci hendrerit id. Curabitur vitae vulputate odio. Nulla facilisi.</p>

                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut quis sapien diam. Sed sagittis, sapien eget porta semper, 
                    sem sem imperdiet augue, eget elementum enim turpis a lorem. Praesent id nisi odio. Aliquam commodo nibh vitae quam 
                    pharetra ultricies. Praesent et vehicula ante. Nunc at erat libero. Nullam quis cursus nisl. Maecenas placerat auctor leo, vel 
                    maximus orci hendrerit id. Curabitur vitae vulputate odio. Nulla facilisi.</p>
                </div>
            </Container>
        </section>
    )
}

export default About;
