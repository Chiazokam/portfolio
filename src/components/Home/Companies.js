import React from 'react';
import Container from '@material-ui/core/Container';
import Paper from '@material-ui/core/Paper';
import Grid from '@material-ui/core/Grid';

const Companies = () => {
    return (
        <section className='companies-section'>
            <Container fixed>
                <div className='section-title'>Who I Have Worked With...</div>
                <div className='companies-content'>

                        <Grid container spacing={4} className='company-card'>
                            <Grid item md={4} sm={12} xs={12}>
                                <Paper>
                                        <p>Andela</p>
                                </Paper>
                            </Grid>
                            <Grid item md={4} sm={12} xs={12}>
                                <Paper>
                                        <p>Punterclash</p>
                                </Paper>
                            </Grid>
                            <Grid item md={4} sm={12} xs={12}>
                                <Paper>
                                        <p>UBA</p>
                                </Paper>
                            </Grid>
                        </Grid>
                </div>
            </Container>
        </section>
    )
}

export default Companies;
