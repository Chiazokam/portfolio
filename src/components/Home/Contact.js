import React from 'react';
import Container from '@material-ui/core/Container';
import PhoneIcon from '@material-ui/icons/Phone';
import LinkedInIcon from '@material-ui/icons/LinkedIn';
import EmailIcon from '@material-ui/icons/Email';
import MessageIcon from '@material-ui/icons/Message';

const Contact = () => {
    return (
        <section className='contact-section'>
            <Container fixed>
                <div className='section-title'>How To Reach Me...</div>
                <div className='contact-content'>
                    <div className="contact-icon bounce">
                        <PhoneIcon />
                    </div>
                    <div className='contact-detail'>
                        <p>&nbsp;+234 0 7016020438</p>
                    </div>

                    <div className="contact-icon bounce">
                        <MessageIcon />
                    </div>
                    <div className='contact-detail'>
                        <p className='direct-message'>&nbsp;Click to send a direct message</p>
                    </div>

                    <div className="contact-icon bounce">
                        <EmailIcon />
                    </div>
                    <div className='contact-detail'>
                        <p>&nbsp;chiazokamecheta@gmail.com</p>
                    </div>

                    <div className="contact-icon bounce">
                        <LinkedInIcon />
                    </div>
                    <div className='contact-detail'>
                        <p>&nbsp;<a href='https://www.linkedin.com/in/chiazokamecheta/' target='_blank'>Click to Visit</a></p>
                    </div>
                </div>
            </Container>
        </section>
    )
}

export default Contact;
