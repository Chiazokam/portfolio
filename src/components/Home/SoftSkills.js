import React from 'react';
import Container from '@material-ui/core/Container';
import Paper from '@material-ui/core/Paper';
import Grid from '@material-ui/core/Grid';
import team from '../../images/team.svg';
import scalable from '../../images/ruler.svg';
import extend from '../../images/extend.svg';
import feedback from '../../images/feedback.svg';
import test from '../../images/test.svg';

const SoftSkills = () => {
    return (
        <section className='soft-skills-section'>
            <Container fixed>
                <div className='section-title'>How I Work...</div>
                <div className='soft-skills-content'>
                        <Grid container spacing={4}>
                            <Grid item md={3} sm={6} xs={12}>
                                <Paper>
                                    <img src={team} alt='team' />
                                    <div className='soft-skill-title'>Good Team Player</div>
                                    <div className='soft-skill-content'>
                                        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut quis sapien diam. 
                                            Sed sagittis, sapien eget porta semper, sem sem imperdiet augue</p>
                                    </div>
                                </Paper>
                            </Grid>
                            <Grid item md={3} sm={6} xs={12}>
                                <Paper>
                                    <img src={scalable} alt='scalable' />
                                    <div className='soft-skill-title'>Scalability</div>
                                    <div className='soft-skill-content'>
                                        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut quis sapien diam. 
                                            Sed sagittis, sapien eget porta semper, sem sem imperdiet augue</p>
                                    </div>
                                </Paper>
                            </Grid>
                            <Grid item md={3} sm={6} xs={12}>
                                <Paper>
                                    <img src={test} alt='test' />
                                    <div className='soft-skill-title'>Test-Driven Development</div>
                                    <div className='soft-skill-content'>
                                        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut quis sapien diam. 
                                            Sed sagittis, sapien eget porta semper, sem sem imperdiet augue</p>
                                    </div>
                                </Paper>
                            </Grid>
                            <Grid item md={3} sm={6} xs={12}>
                                <Paper>
                                    <img src={extend} alt='extend' />
                                    <div className='soft-skill-title'>Extensibility</div>
                                    <div className='soft-skill-content'>
                                        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut quis sapien diam. 
                                            Sed sagittis, sapien eget porta semper, sem sem imperdiet augue</p>
                                    </div>
                                </Paper>
                            </Grid>

                            <Grid item md={3} sm={6} xs={12}>
                                <Paper>
                                    <img src={feedback} alt='feedback' />
                                    <div className='soft-skill-title'>Feedback</div>
                                    <div className='soft-skill-content'>
                                        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut quis sapien diam. 
                                            Sed sagittis, sapien eget porta semper, sem sem imperdiet augue</p>
                                    </div>
                                </Paper>
                            </Grid>
                            <Grid item md={3} sm={6} xs={12}>
                                <Paper>
                                    <img src={extend} alt='extend' />
                                    <div className='soft-skill-title'>Good Team Spirit</div>
                                    <div className='soft-skill-content'>
                                        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut quis sapien diam. 
                                            Sed sagittis, sapien eget porta semper, sem sem imperdiet augue</p>
                                    </div>
                                </Paper>
                            </Grid>
                            <Grid item md={3} sm={6} xs={12}>
                                    <Paper>
                                        <img src={extend} alt='extend' />
                                        <div className='soft-skill-title'>Good Team Spirit</div>
                                        <div className='soft-skill-content'>
                                        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut quis sapien diam. 
                                            Sed sagittis, sapien eget porta semper, sem sem imperdiet augue</p>
                                    </div>
                                </Paper>
                            </Grid>
                            <Grid item md={3} sm={6} xs={12}>
                                <Paper>
                                    <img src={test} alt='test' />
                                    <div className='soft-skill-title'>Good Team Spirit</div>
                                    <div className='soft-skill-content'>
                                        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut quis sapien diam. 
                                            Sed sagittis, sapien eget porta semper, sem sem imperdiet augue</p>
                                    </div>
                                </Paper>
                            </Grid>
                        </Grid>
                </div>
            </Container>
        </section>
    )
}

export default SoftSkills;
