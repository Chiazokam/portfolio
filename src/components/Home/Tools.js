import React from 'react';
import Container from '@material-ui/core/Container';

const Tools = () => {
    return (
        <section className='tools-section'>
            <Container fixed>
                <div className='section-title'>What I Have Worked With...</div>
            </Container>
        </section>
    )
}

export default Tools;
