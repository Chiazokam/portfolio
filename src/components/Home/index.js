import React from 'react';
import KeyboardArrowDownIcon from '@material-ui/icons/KeyboardArrowDown';
import Header from '../Header';
import About from './About';
import Tools from './Tools';
import SoftSkills from './SoftSkills';
import Companies from './Companies';
import Contact from './Contact';

const HomeComponent = () => {
    return (
        <div className='home-wrapper'>
                <section className='home-showcase'>
                    <Header />
                    <div className='showcase-content'>
                        <div className='home-text'>
                            <h1>CHIAZOKAM ECHETA</h1>
                            <p>Speaking passion? speak with me...</p>
                            <div className="arrow bounce">
                                <KeyboardArrowDownIcon />
                            </div>
                        </div>
                    </div>
                </section>
                
                <About />
                <Tools />
                <SoftSkills />
                <Companies />
                <Contact />
        </div>
    );
}

export default HomeComponent;
