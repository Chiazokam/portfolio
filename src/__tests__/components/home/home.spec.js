import React from 'react';
import { shallow } from 'enzyme';
import Home from '../../../components/Home';

describe('Home component', () => {
    it('renders successfully', () => {
    const wrapper = shallow(<Home />);
    expect(wrapper).toMatchSnapshot();
    })
});
