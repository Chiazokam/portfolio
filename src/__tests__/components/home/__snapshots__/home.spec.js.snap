// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component renders successfully 1`] = `ShallowWrapper {}`;
